package/components/editor-utils.tsx

```diff
@@ -788,7 +880,7 @@ export const ScriptsPopup = ({
             variant="ghost"
             onClick={() => {
               option.command();
-              setToolVisibility(IEditorTool.NONE);
+              // setToolVisibility(IEditorTool.NONE);
```

Comment:
Uncomment this out, or remove it with the props so that the build won't complain